jsonschema/validators.py

```diff
@@ -26,6 +26,7 @@
 
 validators = {}
 meta_schemas = _utils.URIDict()
+vocabulary_schemas = _utils.URIDict()
```

Comment:
```suggestion
_VOCABULARIES = _utils.URIDict()
```

Reply:
Also changed to your recomandation :slightly_smiling_face: